core/src/provider.rs

```diff
@@ -596,12 +556,6 @@ impl JsonRpcCachedProvider {
         Ok(filtered_logs)
     }
 
-    #[tracing::instrument(skip_all)]
-    pub async fn get_chain_id(&self) -> Result<U256, ProviderError> {
```

Comment:
hiding this as should be never called by our code as affects negatively indexing speed